styles.css

```diff
@@ -0,0 +1,14 @@
+/* tool_excimer styles */
+
+.path-admin-tool-excimer-index #loading {
+  text-align: center;
```

Comment:
I think there is a code standard for 4 spaces for css, think thats the same as php and everything else